pandas/_libs/tslibs/fields.pyx

```diff
@@ -96,15 +97,11 @@ def get_date_name_field(ndarray[int64_t] dtindex, object field):
         pandas_datetimestruct dts
         int dow
 
-    _dayname = np.array(
-        ['Monday', 'Tuesday', 'Wednesday', 'Thursday',
-            'Friday', 'Saturday', 'Sunday'],
-        dtype=np.object_)
-
     count = len(dtindex)
     out = np.empty(count, dtype=object)
 
     if field == 'weekday_name':
```

Comment:
should'nt this be removed?

Reply:
We discussed just depreciating `weekday_name` for now (and removing it in a future version)

Reply:
see my commeht below, this is *the same* implementation as ``day_name``, no? no reason to duplicate it

Reply:
Ah I see. Yes, I'll try to remove this since they are nearly identical.